code/game/objects/items/spear.dm

```diff
@@ -161,6 +161,13 @@
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
 
+/obj/item/spear/Initialize()
+	. = ..()
+	if(!(istype(get_area(src), /area/tdome/arena)))
+		visible_message("<span class='warning'>...Just kidding. [src] snaps in two.</span>")
```

Comment:
lol

Reply:
okay this I didn't know, that's hilarious